mod_reforged/hooks/config/tip_of_the_day.nut

```diff
@@ -24,6 +24,8 @@ for (local index = (::Const.TipOfTheDay.len() - 1); index >= 0; index--)
     "Morale Checks are harder for every adjacent enemy and easier for every adjacent ally.",
     "Click on an active contract to focus on its target on the world map, if it is known to you.",
     "The more mods you add the harder it is to debug the game for the modder.",
-    "Tavern rumors will never be about legendary locations you have already discovered"
+	"Tavern rumors will never be about legendary locations you have already discovered.",
+	"Enemies will only drop loot if your faction has dealt atleast 50% damage to them.",
```

Comment:
Enemies will only drop loot if your faction has dealt at least 50% of the total damage received by that enemy.